libs/designer/src/lib/ui/panel/panelRoot.tsx

```diff
@@ -45,16 +46,21 @@ export const PanelRoot = (props: PanelRootProps): JSX.Element => {
 
   const dismissPanel = useCallback(() => dispatch(clearPanel()), [dispatch]);
 
-  const commonPanelProps: CommonPanelProps = useMemo(
-    () => ({
+  const commonPanelProps: CommonPanelProps = useMemo(() => {
+    let customLocation = undefined;
```

Comment:
```suggestion
    let customLocation = customPanelLocations?.find(x => currentPanelMode === x.panelMode);
```

then you can remove the entire loop below